.github/workflows/cicd-demo.yml

```diff
@@ -40,8 +40,8 @@ jobs:
       - name: Download CodeGuru Reviewer CLI
         shell: bash
         env:
-          VERSION: 0.1.0
-        run: curl -OL "https://github.com/martinschaef/aws-codeguru-cli/releases/download/$VERSION/aws-codeguru-cli.zip"
+          VERSION: 0.2.1
```

Comment:
nit for the future: could this be scraped from the build.gradle file to avoid the redundancy going out of sync?

Reply:
Hmm ... not sure how; we want to make that easy to copy. Any suggestions?

Reply:
I would add a `VERSION` file with nothing but the version, which is trivial to parse, and use that everywhere else.